### PR TITLE
Highcharts updated to version 8.0.0

### PR DESCRIPTION
**Pull request by SilverGull**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x ] Tests for the changes have been added (for bug fixes / features)
- [x ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [x] Other... Please describe: HighCharts library update

## What is the current behavior?

HighCharts is at version 7.0.3

Issue Number: N/A

## What is the new behavior?
HighCharts has been updated from version 7.0.3 to 8.0.0
A bit of cleanup in the example code

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/kirby/components/chart/options/activitygauge.ts [hunk 1]
```diff
@@ -10,9 +10,11 @@ export const ActivityGaugeOptions: Options = {
     },
     backgroundColor: 'transparent',
     type: 'solidgauge',
-    description: '',
     height: '110%',
   },
+  accessibility: {
+    description: '',
```
Comment (askarby) [inline src/kirby/components/chart/options/activitygauge.ts]: This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?

File: src/kirby/components/chart/options/areaspline.ts [hunk 2]
```diff
@@ -10,6 +10,9 @@ export const AreaSplineOptions: Options = {
     margin: 0,
     marginTop: 48,
   },
+  accessibility: {
```
Comment (askarby) [inline src/kirby/components/chart/options/areaspline.ts]: This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?

File: src/kirby/components/chart/options/donut.ts [hunk 3]
```diff
@@ -10,6 +10,8 @@ export const DonutOptions: Options = {
     },
     backgroundColor: 'transparent',
     type: '',
+  },
+  accessibility: {
```
Comment (askarby) [inline src/kirby/components/chart/options/donut.ts]: This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?

File: src/kirby/components/chart/options/timeseries.ts [hunk 4]
```diff
@@ -16,6 +16,9 @@ export const TimeSeriesOptions: Options = {
     margin: 0,
     marginTop: 48,
   },
+  accessibility: {
```
Comment (askarby) [inline src/kirby/components/chart/options/timeseries.ts]: This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?

File: src/app/showcase/chart-showcase/chart-showcase.component.html [hunk 5]
```diff
@@ -12,13 +12,17 @@ <h4>Properties:</h4>
       <tbody>
         <tr>
           <td><code>height</code></td>
-          <td>The height of the chart in px for web and dp for native</td>
-          <td><code>number</code></td>
+          <td>
+            If a number, the height is given in pixels. If given a percentage string (for example
```
Comment (jakobe) [inline src/app/showcase/chart-showcase/chart-showcase.component.html:16]: Does this also allow a pixel value, e.g. `300px` and/or `em`s? If yes, please add to description.

File: src/kirby/components/chart/chart.component.spec.ts [hunk 6]
```diff
@@ -65,13 +65,13 @@ describe('ChartComponent', () => {
   });
 
   it('should have dataLabels enabled as default', () => {
-    expect(component.options.plotOptions.pie.dataLabels.enabled).toBe(true);
+    expect(component.options.plotOptions.series.dataLabels).toEqual({ enabled: true });
```
Comment (jakobe) [inline src/kirby/components/chart/chart.component.spec.ts:68]: Why not as before?
```suggestion
    expect(component.options.plotOptions.series.dataLabels.enabled).toBeTrue();
```

File: src/kirby/components/chart/chart.component.spec.ts [hunk 7]
```diff
@@ -65,13 +65,13 @@ describe('ChartComponent', () => {
...
   });
 
   it('should disable dataLabels when false', () => {
     component.showDataLabels = false;
     component.ngOnInit();
-    expect(component.options.plotOptions.pie.dataLabels.enabled).toBe(false);
+    expect(component.options.plotOptions.series.dataLabels).toEqual({ enabled: false });
```
Comment (jakobe) [inline src/kirby/components/chart/chart.component.spec.ts:74]: Why not as before?
```suggestion
    expect(component.options.plotOptions.series.dataLabels.enabled).toBeFalse();
```

File: src/kirby/components/chart/chart.component.ts [hunk 8]
```diff
@@ -83,16 +83,18 @@ export class ChartComponent implements OnInit, OnChanges {
...
     if (this.options.chart) {
       this.options.chart.height = this.height;
-      this.options.chart.description = this.description;
+      if (this.options.accessibility) {
+        this.options.accessibility.description = this.description;
+      }
       switch (this.options.chart.type) {
         case ChartType.PIE:
-          this.options.plotOptions.pie.dataLabels.enabled = this.showDataLabels;
+          this.options.plotOptions.series.dataLabels = { enabled: this.showDataLabels };
```
Comment (jakobe) [inline src/kirby/components/chart/chart.component.ts:91]: Why not as before?

File: src/kirby/components/chart/chart.component.spec.ts (hunk 6 above)
Comment (lydemann) [reply src/kirby/components/chart/chart.component.spec.ts:68]: Interface change in Highcharts 8

Comment (lydemann) [reply src/kirby/components/chart/chart.component.spec.ts:68]: Changed to pie.dataLabels as before, but still need the typecast

File: src/app/showcase/chart-showcase/chart-showcase.component.html (hunk 5 above)
Comment (lydemann) [reply src/app/showcase/chart-showcase/chart-showcase.component.html:16]: I haven't changed anything to the existing interface.

**Comment by lydemann** [comment]
Closing this as it is implemented here: https://github.com/kirbydesign/designsystem/pull/1022

**Pull request closed without merging**